metagraph/src/cli/config/config.hpp

```diff
@@ -40,8 +40,9 @@ class Config {
     bool count_kmers = false;
     bool print_signature = false;
     bool query_presence = false;
+    bool query_counts = false;
     bool query_coords = false;
-    bool verbose_coords = false;
+    bool verbose_output = false;
```

Comment:
I think `verbose_coords` was more descriptive, `--verbose-output` may be confused with `--verbose`

Reply:
Now it's both `verbose_coords` and `verbose_counts`, so I had to rename it. Not a major thing anyway.